src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -1128,7 +1136,7 @@ private void ValidateUriParameterValue(Uri uri, string parameterName)
         {
             Dbg.Assert(!String.IsNullOrWhiteSpace(parameterName), "parameterName should not be null or whitespace");
 
-            if (uri != null && !Uri.IsWellFormedUriString(uri.ToString(), UriKind.Absolute))
+            if (uri != null && !Uri.IsWellFormedUriString(uri.AbsoluteUri, UriKind.Absolute))
```

Comment:
I prefer `null` on the left, but most of the code in this file has it on the right, I'll change my usage to make it consistent